framework/helpers/BaseConsole.php

```diff
@@ -340,6 +340,93 @@ public static function ansiStrlen($string)
         return mb_strlen(static::stripAnsiFormat($string));
     }
 
+    //
+    // Exit code constant names, values, and descriptions are taken from
+    // OpenBSD's sysexits(3) man page http://man.openbsd.org/sysexits
+    //
+    // The command completed successfully.
+    const EX_OK = 0;
```

Comment:
These exit codes are used exclusively in console controller so I think that console controller should define them.